package.json

```diff
@@ -17,26 +17,28 @@
   "scripts": {
     "commitlint": "commitlint",
     "commitmsg": "commitlint -e $GIT_PARAMS",
-    "lint": "eslint --cache lib test",
+    "lint": "eslint --cache lib test && npm run type-lint",
```

Comment:
apparently prettierrc told my prettier to update these indentations :/ I can revert if a problem

Reply:
yeah that should be handled by the editor and .editorconfig for json files. revert that please.